### ISSUE TITLE
Sample

### ISSUE THREAD

**Issue by shahroq** [tareq1988/wp-generators#9]
Hi,
Can you plz provide a sample of generated code? It can be upload into a /demo/ folder of this repo.
Thanx,

**Comment by matichek** [comment]
Yes, that would be very nice, cause current youtube is missing some key info and code has errors. At least upload "plugins.php" to see how classes are activated. Thank you.
